ovirt_imageio/client/_options.py

```diff
@@ -78,19 +140,54 @@ def config_file(self):
         return os.path.join(base_dir, "ovirt-img.conf")
 
     def _merge_config(self, args):
+        """
+        Read options from specified config section and set the value for unused
+        command line options. This must be done after parsing command line
+        options since we depend on --config.
+        """
         config = configparser.ConfigParser(interpolation=None)
         config.read([self.config_file])
 
         if not config.has_section(args.config):
             self._parser.error(
                 f"No section: '{args.config}' in '{self.config_file}'")
 
-        for name in ("engine_url", "username", "password", "cafile"):
-            if getattr(args, name) is not None:
+        for option in self._OPTIONS:
```

Comment:
That's a nice and clear way, to fill in the config-file-arguments after the CLI-arguments!